src/renderer/components/GameTeam/QuessYearQuestionTeam/GuessYearQuestionTeam.tsx

```diff
@@ -1,36 +1,43 @@
 import styles from './GuessYearQuestionTeam.scss';
 import { useSelector } from 'react-redux';
-import { selectCurrentQuestion, selectTeamScoredId } from '../../../../redux/gameSlice';
+import { selectCurrentQuestion } from '../../../../redux/gameSlice';
 
 import Button from '../../Button/Button';
 import React, { useState } from 'react';
 import { Slider } from '@mui/material';
+import { selectCurrentGameId, selectCurrentShowId, selectUser } from '../../../../redux/userSlice';
+import { SaveTeamAnswerRequest } from '../../../../types';
+import { sendMessage } from '../../../../redux/websocketSlice';
 
 function GuessYearQuestionTeam() {
-  const [winningScore, setWinningScore] = useState(1950);
   const { question } = useSelector(selectCurrentQuestion);
+  const currentShowId = useSelector(selectCurrentShowId);
+  const currentGameId = useSelector(selectCurrentGameId);
+  const currentUser = useSelector(selectUser);
+
+  const [winningScore, setWinningScore] = useState(1950);
```

Comment:
I think, that for each question of time GuessYear, we should provide bottom and up boundaries for year selection, and based on them calculate the middle value and provide it as initial state. Because each question can have different boundaries, right? It should be defined on the data of question.